### ISSUE TITLE
bug/scrollable elements with tabindex=0 lack role="region" and name

### ISSUE THREAD

**Issue by CorinaMurg** [CorinaMurg/BugReport-WebForEveryone#7]
Horizontal scrolling `<code>` elements have  `tabindex="0"` so they are keyboard accessible.

To do: mark them as regions with an accessible name, otherwise screen readers will announce their entire content.
